src/ast.c

```diff
@@ -9,17 +9,24 @@
   usz alignment;
   span name;
   bool is_signed;
+  /// A unique ID that is used to compare primitives.
   uint8_t id;
 } TypePrimitive;*/
 static Type t_void_def = {
-    .kind = TYPE_PRIMITIVE,
-    .source_location = {0},
-    .primitive = {
-        .size = 0,
-        .alignment = 0,
-        .name = literal_span_raw("void"),
-        .id = 0,
-    },
+  .kind = TYPE_PRIMITIVE,
+  .source_location = {0},
+  .primitive = {
+    .size = 0,
+    .alignment = 0,
+    .name = literal_span_raw("void"),
+    .id = 0,
+  },
+};
+
+static Type t_pointer_def = {
```

Comment:
Do you also recommend changing `t_pointer`? If not, this is implemented as of ec7b772f4fd54939191c334be7b844aed1bb8847

Reply:
Yeah I meant changing both; figured that was kinda implied since no-one cares about the `_def` since it’s a local symbol and I didn’t want to type that much haha